httpx/dispatch/http2.py

```diff
@@ -172,15 +194,9 @@ def is_connection_dropped(self) -> bool:
 
 
 class HTTP2Stream:
-    def __init__(
-        self,
-        stream_id: int,
-        connection: HTTP2Connection,
-        state: h2.connection.H2Connection,
-    ) -> None:
+    def __init__(self, stream_id: int, connection: HTTP2Connection,) -> None:
```

Comment:
Formatting nit (I've already had these, not sure why Black doesn't reformat them):

```suggestion
    def __init__(self, stream_id: int, connection: HTTP2Connection) -> None:
```

Reply:
Thanks!

Reply:
https://github.com/psf/black#trailing-commas was changed recently, I believe https://github.com/psf/black/pull/826 was the ~culprit~ responsible PR. There appears to be a follow-up PR https://github.com/psf/black/pull/1164 to remedy it.